types/ftps/ftps-tests.ts

```diff
@@ -0,0 +1,24 @@
+import * as FTP from 'ftps';
```

Comment:
Generally speaking, you can't instantiate the symbol imported using `* as x`.  Did you mean `import FTP = require('ftps');`?

Reply:
Yeap that was what I meant :smile: 
Just made the change, thank you for your review!